File: examples/db/listTables/go/listTables.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/db"
)

// List tables in the DB
func ListTables() {
	dbService := db.NewDbService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := dbService.ListTables(&db.ListTablesRequest{})
	fmt.Println(rsp, err)
}

File: examples/db/listTables/node/listTables.js
const { DbService } = require("micro-js-client/db");

// List tables in the DB
async function listTables() {
  let dbService = new DbService(process.env.MICRO_API_TOKEN);
  let rsp = await dbService.listTables({});
  console.log(rsp);
}

listTables();
